app/src/main/res/values/strings.xml

```diff
@@ -240,4 +235,13 @@
     <string name="confirm_transaction_dialog_message">You are about to confirm the transaction with your currently imported owner key. This confirmation is off-chain. The transaction should be executed separately in the web interface.</string>
     <string name="confirmation_successfully_submitted">Confirmation successfully submitted</string>
     <string name="ens_name_contains_illegal_character">Name contains invalid characters</string>
+
+    <string name="default_fallback_handler">DefaultFallbackHandler</string>
+    <string name="unknown_fallback_handler">Unknown</string>
+    <string name="empty_string" translatable="false" />
```

Comment:
I think that's overkill. Why not just use ""

Reply:
"" is not an Int

Reply:
We could use null as in `Int?`